server/testbed/mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2022 ANSSI
+# SPDX-License-Identifier: Apache-2.0
+
+[Distribution]
+Distribution=debian
+Release=testing
+
+[Packages]
```

Comment:
No clue if there is a concept of "run package" for mkosi, but fyi I had to install `swtpm` to get this command (used in the Makefile) on Debian.